docs/byo-mgmt-gitops.md

```diff
@@ -60,7 +60,9 @@ On each of the edge clusters, you can enable [monitoring for user-defined projec
 
 If you forward metrics from each edge cluster to a central location, you can find a list of example metrics from the
 OpenVINO and Seldon model servers from our examples in
-[metrics_list](../acm/odh-core/acm-observability/files/metrics_list.yaml) that you may wish to forward.
+[metrics_list](../acm/odh-core/acm-observability/files/uwl_metrics_list.yaml) that you may wish to forward.
+
+NOTE: If you are not deploying our example models then you must edit the [static list](../acm/odh-edge/apps/telemetry/otel-collector.yaml) of targets to contain the service pointing to your custom model.
```

Comment:
I think this NOTE is applicable to all scenarios, rather than just the "bring your own management" one, right? I think this should be moved to some more primary location, like the README.md. WDYT?

Reply:
Yep that makes sense.